README.md

```diff
@@ -68,7 +68,7 @@ pip install -e .
 
 ### Integration tests
 
```

Comment:
It looks like there are no longer integration tests so I think we can remove this section entirely.
```suggestion
```